pypit_files/keck_lris_blue_long_400_3400_d560.pypit

```diff
@@ -1,31 +1,30 @@
-# This is a comment line
+# Parameters
+[rdx]
+    spectrograph = keck_lris_blue
+    sortroot = lris_blue_long_400_3400_d560
 
-# Change the default settings
-run ncpus 1
-run spectrograph keck_lris_blue
-output verbosity 2
-output overwrite True
-output sorted lris_blue_long_400_3400_d560
+[calibrations]
+    [[biasframe]]
+        useframe = overscan
+        [[[process]]]
+            sigrej = 20.
+    [[pixelflatframe]]
+        number = 1
+    [[traceframe]]
+        number = 3
+    [[flatfield]]
+        frame = /home/xavier/Projects/Python/PYPIT-development-suite/CALIBS/PYPIT_LRISb_pixflat_B400_2x2_15apr2015.fits.gz
```

Comment:
Do you think it is possible to make this path more generalised?

Reply:
Yeah, I think so, but I haven't worked out the details yet.  This is raised as [PYPIT #402](https://github.com/PYPIT/PYPIT/issues/402).  I haven't set a milestone for this, but we could make it 1.0.